azext_iot/sdk/service/models/x509_thumbprint.py

```diff
@@ -1,32 +0,0 @@
-# coding=utf-8
-# --------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
-# --------------------------------------------------------------------------
-
-from msrest.serialization import Model
-
-
-class X509Thumbprint(Model):
```

Comment:
why did we get rid of all the models?

Reply:
These models were for the old service SDK which nothing uses anymore after this PR -- there are "refreshed" models (as in refreshed for the IoT Hub API version 2019-10-01, in the sdk/iothub/service directory.